packages/api/src/surveys/survey.repository.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@nestjs/common';
+
+import type { Survey, UserSurvey } from '@coderscamp/shared/models';
+
+import { SurveyRepositoryPort } from '../contracts/survey.repository';
+
+@Injectable()
+export class SurveyRepository {
```

Comment:
I'm not sure what is the purpose of this class? 
Why not make implementations of SurveyRepositoryPort @Injectable?